pkgs/linux-jovian/default.nix

```
@@ -17,7 +17,7 @@ buildLinux (args // rec {
   # branchVersion needs to be x.y
   extraMeta.branch = versions.majorMinor version;
 
-  kernelPatches = [
+  kernelPatches = (lib.optionals (args ? kernelPatches) args.kernelPatches) ++ [
```

Mostly stylistic, but this works:

```suggestion
  kernelPatches = (args.kernelPatches or []) ++ [
```